### PR TITLE
feat: add an auto selected volume in creating and cloning a data folder

### PR DESCRIPTION
**Pull request by agatha197**
- resolves https://github.com/lablup/giftbox/issues/502
- Add 'auto' vhost when creating or cloning a data folder. (Auto volume is added only when there are multiple hosts.)
- The auto-selected folder is the lowest usage folder.
- If the usage percentage of the vhost is empty, set auto-selected vhost to the default folder.

## Screenshots
- create `autoSelectedVhostFolder` folder with auto host.
image
- The auto-selected folder is set to the ceph which is the lowest usage.
image
- But after creating a new folder, the original host name (ceph) will be displayed on the storage list.
image
- Info description. The usage info will follow the original host's (ceph).
image

### CODE REVIEW

File: resources/storage_metadata.json [hunk 1]
```diff
@@ -84,6 +84,11 @@
         "SpectrumScale",
         "GPFS"
       ]
+    },
+    "auto": {
+      "name": "Auto",
+      "description": "Automatically select the host with the least usage among the accessible folders for users. This prevents the folders from being concentrated on a single host.",
```
Comment (adrysn) [inline resources/storage_metadata.json]: Let's rewrite:
> Choose the host with the lowest usage from the available host volumes for users. This ensures that usage is evenly distributed across multiple storage volumes, avoiding over-concentration on any single volume.

**Review by adrysn — CHANGES_REQUESTED** [review]
Let's specify which volume is automatically selected like the image below:
image

**Comment by agatha197** [comment]
> Let's specify which volume is automatically selected like the image below: image

Thanks for the feedback. That's a great idea. I reflected as below.
image

**Review by adrysn — COMMENTED** [review]
LGTM.

**Review by adrysn — APPROVED** [review]

**Pull request merged**